pageserver/src/tenant/timeline.rs

```diff
@@ -426,6 +426,11 @@ pub struct Timeline {
 
     /// Indicate whether aux file v2 storage is enabled.
     pub(crate) last_aux_file_policy: AtomicAuxFilePolicy,
+
+    #[cfg(test)]
+    /// Some test cases directly place keys into the timeline without actually modifying the directory
+    /// keys (i.e., DB_DIR). Test cases will put such keyspaces here.
```

Comment:
```suggestion
    /// Some test cases directly place keys into the timeline without actually modifying the directory
    /// keys (i.e., DB_DIR). Test cases will put such keyspaces here.
    #[cfg(test)]
```

Please always order them this way around.